book/03-git-branching/sections/rebasing.asc

```diff
@@ -126,95 +126,73 @@ Ahh, ngunit ang kaligayahan ng pag-rebase ay mayroon ding kakulangan, na maaarin
 
 Kung susundin mo ang patnubay na iyon, magiging maayos ka. Kung hindi, kasusuklaman ka ng mga tao, at kakamuhian ka ng iyong mga kaibigan at pamilya.
 
-When you rebase stuff, you're abandoning existing commits and creating new ones that are similar but different.
-If you push commits somewhere and others pull them down and base work on them, and then you rewrite those commits with `git rebase` and push them up again, your collaborators will have to re-merge their work and things will get messy when you try to pull their work back into yours.
+Kapag ikaw ay nag-rebase ng mga bagay, ikaw ay lumilisan sa umiiral na mga commit at gumagawa ng mga panibago na kahawig ngunit naiiba. Kung magpu-push ka ng mga commit saanman at ang iba ay magpu-pull down ng mga ito at magbabase sa trabaho nito, at pagkatapos ay isusulat mo muli ang mga commit na iyon gamit ang `git rebase` at ipu-push up ang mga ito ulit, ang iyong mga katulong ay kailangang mag merge muli ng kanilang trabaho at ang mga bagay ay magugulo kapag sinubukan mong mag-pull sa kanilang trabaho patungo pabalik sa iyo.
```

Comment:
* Insert line break before ``Kung magpu-push ka ng mga commit saanman``
* ``ulit`` to ``uli`` or ``muli`` -> `i-push muli ang mga ito`?

Reply:
changing it now